### PR TITLE
WT-8015 Removing Product should only send ISBN and should be on its own

### PR DESCRIPTION
**Pull request by sachinmaharjan**
Story: https://blurb-books.atlassian.net/browse/WT-8015
Previously it was using XML creation for PRODUCT_LIST
Removing Product should only send ISBN and should be on its own

### CODE REVIEW

**Comment by sachinmaharjan** [comment]
@pierrel @nhullrichter-blurb

File: spec/ruby-mws/api/feed_spec.rb [hunk 1]
```diff
@@ -425,6 +439,32 @@
         end
       end
 
+      context "#product_list_remove" do
+        it 'should be able to set the inventory'  do
+          response = mws.feeds.submit_feed(MWS::API::Feed::PRODUCT_LIST, product_list_remove_hash_list)
+          response.feed_submission_info.should_not be_nil
```
Comment (nhullrichter-blurb) [inline spec/ruby-mws/api/feed_spec.rb]: This is implicitly tested below, you don't need to test it here.

**Comment by nhullrichter-blurb** [comment]
+1 after comment

**Comment by pierrel** [comment]
nice
+1

File: spec/ruby-mws/api/feed_spec.rb (hunk 1 above)
Comment (sachinmaharjan) [inline spec/ruby-mws/api/feed_spec.rb]: removed

**Comment by sachinmaharjan** [comment]
merged

**Pull request merged**